xla/service/gpu/backend_configs.proto

```diff
@@ -104,6 +104,7 @@ message GemmBackendConfig {
 
   optional bool grad_x = 16;
   optional bool grad_y = 17;
+  optional bool has_damax_output = 18;
```

Comment:
Why is this an optional bool? Do we need three different states?

Reply:
Without optional, it will require protobuf to initialize the has_damax_output to false. though protobuf does it, but I think have an optional state should make the code more clear.